spec/layout_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'spec_helper'
+
+RSpec.describe 'layout' do
+    let(:page) { Nokogiri::HTML(file_at('/')) }
+
+    it 'has correctly configured system-status meta tag' do
+        meta = page.at_css('meta[name="system-status"]')
+        expect(meta[:content]).to eq('Login.gov site up and running')
+    end
+end
```

Comment:
This project apparently lacks Rubocop linting altogether, but in the meantime there's some formatting issues we'll want to fix up here:

- Indent by two spaces
- End-of-file newline

```suggestion
RSpec.describe 'layout' do
  let(:page) { Nokogiri::HTML(file_at('/')) }

  it 'has correctly configured system-status meta tag' do
    meta = page.at_css('meta[name="system-status"]')
    expect(meta[:content]).to eq('Login.gov site up and running')
  end
end

```